operator/cmd/mesh/testdata/manifest-generate/data-snapshot/profiles/preview.yaml

```diff
@@ -10,9 +10,9 @@ spec:
       proxyMetadata:
         # Enable Istio agent to handle DNS requests for known hosts
         # Unknown hosts will automatically be resolved using upstream dns servers in resolv.conf
-        ISTIO_META_DNS_CAPTURE: "ALL"
```

Comment:
isn't this backwards incompatible? people will probably have it set explicitly

Reply:
It is from #26954, introduced recently not in 1.7